data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+				choice
+					`	(Try to convey that I intend no harm.)`
+						goto peace
+					`	(Drop the repair equipment and run back into my ship.)`
+						goto flee
+					`	(Try to make a non-lethal shot at the one behind the turret and run.)`
+				`	You hit the creature manning the turret, causing it to explode with a strident sound. Before you can enjoy your short-lived victory, you notice a motion further down the hallway. Turning your head you see a gunner flashing in a deep red, before shooting you down and avenging their fallen comrade.`
+					die
+				label peace
+				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. Both sides are still for a moment until one of the aliens slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate. The other four back off in response. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`
+				`	With the repair equipment delivered, there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`
+					goto end
+				label flee
+				`	You drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look of one of the aliens: blue and yellow tones are mixing on their skin, but they are not taking any action.`
+				label end
+				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
+				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange, considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
+				action
+					set "incipias: helped the derelict"
+		personality
+			confusion 50
+			derelict staying uninterested
+	on enter "Hui'uc"
+		conversation
+			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`
+
+mission "Incipias: Help The Stranded 1"
+	landing
+	source
+		government "Quarg (Incipias)"
+	destination "Pon'tes"
+	to offer
+		has "incipias: helped the derelict"
+	on offer
+		conversation
+			`Maybe the Quarg could help you with the stranded ship.`
+			choice
+				`	(Tell them about it.)`
+				`	(Not right now.)`
```

Comment:
```suggestion
		conversation
			`Would you like to ask the Quarg for insight on your strange encounter with the stranded alien ship?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
```
In other places in the game, if we have a short defer dialog, we typically ask the player in a form of a question if they want to continue. For example, Hai first contact.